tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -4145,6 +4155,14 @@ class HLSLExternalSource : public ExternalSemaSource {
 
   Sema *getSema() { return m_sema; }
 
+  void BuildCallGraphForEntry(FunctionDecl *EntryFnDecl) {
+    m_callGraph.BuildForEntry(EntryFnDecl);
+  }
+
+  FunctionDecl *CheckForCallGraphRecursion(FunctionDecl *EntryFnDecl) {
```

Comment:
These two methods just wrap methods on `m_callGraph`.  Why not just use `getCallGraph.<method>()` instead?